### PR TITLE
Created Navigation Component

### PR DESCRIPTION
**Pull request by AdityaTeltia**
### Description

Created the navigation component in path ```src/component``` .

### Related Issue
Solved issue #21 

Fixes # [ISSUE]

### Type of Change:

- Code
- User Interface
- New Feature

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)

### How Has This Been Tested?

Tested by Rendering on Local Machine !

### Checklist:

- [x] Pull request is made against `navigationComponent` temporary branch.
- [x] My code follows the style guidelines of this project.
- [x] I run `npm run format` before creating the pull request.
- [x] I have performed a self-review of my own code.

### Screenshots (if appropriate):
[localhost_3000_ (1)](https://user-images.githubusercontent.com/67232537/109935267-6e0a5480-7cf3-11eb-9f15-f35988ceb5ac.png)

### CODE REVIEW

File: src/components/Navigation.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      >
+        <List>
+          {[
+            'Dashboard',
+            'Profiles',
+            'Events',
+            'Blogs',
+            'Settings',
+            'Log out'
+          ].map((text, index) => (
```
Comment (Abhishek-kumar09) [inline src/components/Navigation.js]: export it as a seperate list

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

File: src/components/Navigation.js (hunk 1 above)
Comment (AdityaTeltia) [reply src/components/Navigation.js]: Make individual list items for each right ? ```<ListItem>Dashboard</ListItem>``` and so on and assign the active class to one  and different color .

Comment (Abhishek-kumar09) [reply src/components/Navigation.js]: @AdityaTeltia  No no, just export this array like :
```
const navItems = [
            'Dashboard',
            'Profiles',
            'Events',
            'Blogs',
            'Settings',
            'Log out'
          ]
```
and then use it here like: 
```
navItems.map(....
```

File: src/components/Navigation.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            <ListItem button key={text} style={{ paddingLeft: '20%' }}>
```
Comment (Abhishek-kumar09) [inline src/components/Navigation.js]: also add a `key` prop like `key={index}`

Comment (AdityaTeltia) [reply src/components/Navigation.js]: Ok sure !

File: src/components/Navigation.js (hunk 1 above)
Comment (AdityaTeltia) [reply src/components/Navigation.js]: Oh ok got it

**Comment by AdityaTeltia** [comment]
@Abhishek-kumar09  Go through it once again I have made all the required changes.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -18,9 +18,7 @@ A clear description of what the actual behavior is.
 
 ## Steps to Reproduce the Problem
 
-1.
-2.
-3.
+1. 2. 3.
```
Comment (Abhishek-kumar09) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Why this change? HAve you made it intentionally?

Comment (AdityaTeltia) [reply .github/ISSUE_TEMPLATE/bug_report.md]: No it automatically changed after running ```npm run format```

Comment (Abhishek-kumar09) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Okay I just added prettierignore
https://github.com/codeforcauseorg/causefolio/blob/development/.prettierignore

Please revert the changes by copy pasting the files as it is from the main repo into places where the changes in `.github` files are made. Then rerun the `npm run format` to verify if the prettierignore is properly set up, since I didn't get time to check it personally, if you can do.


Importantly, we don't want to change github issues and PR templates due to prettier.

Comment (AdityaTeltia) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Sure !

**Comment by AdityaTeltia** [comment]
@Abhishek-kumar09 Check now !

**Comment by Abhishek-kumar09** [comment]
Thanks for the PR @AdityaTeltia
